c/tskit/tables.c

```diff
@@ -6831,8 +6831,9 @@ simplifier_init(simplifier_t *self, const tsk_id_t *samples, size_t num_samples,
      * debateable whether we need it. If we remove, we definitely need explicit
      * tests to ensure we're doing sensible things with duplicate sites.
      * (Particularly, re TSK_REDUCE_TO_SITE_TOPOLOGY.) */
-    ret = tsk_table_collection_check_integrity(tables,
-        TSK_CHECK_EDGE_ORDERING | TSK_CHECK_SITE_ORDERING | TSK_CHECK_SITE_DUPLICATES);
+    ret = tsk_table_collection_check_integrity(
+        tables, TSK_CHECK_EDGE_ORDERING | TSK_CHECK_SITE_ORDERING
+                    | TSK_CHECK_SITE_DUPLICATES | TSK_CHECK_INDIVIDUAL_ORDERING);
```

Comment:
Why do we need TSK_CHECK_INDIVIDUAL_ORDERING  this for simplify?

Reply:
Seems I got over zealous! Removed!